src/app/services/user.service.ts

```diff
@@ -0,0 +1,22 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {Observable} from 'rxjs';
+
+import {User} from '../models/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+  baseUrl = 'https://good-deed.herokuapp.com';
```

Comment:
so why can't you reuse the constant from above ? :D

Reply:
Done